### PR TITLE
evy: Add canvas poly builtin

### PR DESCRIPTION
**Pull request by juliaogris**
Add canvas poly builtin to draw polylines and polygons.

New sample:
https://evy-lang--125-ofi7tbe4.web.app/#poly

### CODE REVIEW

File: frontend/module/yace-editor.js [hunk 1]
```diff
@@ -467,14 +467,14 @@ const builtins = new Set([
   "rand",
   "rand1",
   "move",
-  "read",
```
Comment (camh-) [inline frontend/module/yace-editor.js:470]: read is still a builtin, isn't it?

**Review by camh- — APPROVED** [review]
LGTM, even if you do insist on calling it poly.

File: frontend/module/yace-editor.js (hunk 1 above)
Comment (juliaogris) [reply frontend/module/yace-editor.js:470]: yeah, it was a duplicate. I'll break it out into a separate commit (ie should be deleted).

**Comment by juliaogris** [comment]
> LGTM, even if you do insist on calling it poly.

Happy to call it something short and better if you can think of anything.

**Pull request merged**